applications/FluidDynamicsApplication/tests/Cavity/input_test_parameters.json

```diff
@@ -0,0 +1,81 @@
+{
+    "test_settings" : {
+        "execute_solve" : false
```

Comment:
I think we can remove these `"execute_solve"` settings. They appear to have been replaced by `fluid_analysis_without_solution.py` which is called directly in `hdf5_io_test.py`.

Reply:
done :)